tools/serverpod_cli/lib/src/generator/dart/library_generators/model_library_generator.dart

```diff
@@ -49,6 +49,20 @@ class SerializableModelLibraryGenerator {
 
     return Library(
       (libraryBuilder) {
+        if (classDefinition.isSealedTopNode) {
+          for (var child in classDefinition.descendantClasses) {
+            libraryBuilder.directives
+                .add(Directive.part('${child.fileName}.dart'));
+          }
+        }
+
+        if (!classDefinition.isSealedTopNode &&
+            classDefinition.sealedTopNode != null) {
+          libraryBuilder.directives.add(
+            Directive.partOf('${classDefinition.sealedTopNode?.fileName}.dart'),
```

Comment:
Suggestion: Assign sealedTopNode to a variable on the outside and nullcheck it.

This way we remove unecessary case where this value COULD return something different the second time we check it.